server/server.js

```diff
@@ -77,6 +72,40 @@ app.get('/api/products/:productId', async (req, res, next) => {
   }
 });
 
+// GET user's account info
+app.get('/api/tables/public.users', authMiddleware, async (req, res, next) => {
+  try {
+    const sql = `select "firstName", "lastName", "username", "email"
+    from "users"
+    order by "userId"`;
+    const result = await db.query(sql);
+    res.json(result.rows);
+  } catch (err) {
+    next(err);
+  }
+});
+
+// GET orders by the userId
+app.get('/api/orders/:userId', authMiddleware, async (req, res, next) => {
+  try {
+    const userId = req.params.userId;
+    const sql = `
+    select *
+    from "users"
+    join "orders" using "userId"
+    join "orderItems" using "orderId"
+    join "products" using "productId"
+    where "userId" = $1
+    `;
+    const params = [userId];
+    const result = await db.query(sql, params);
+    const user = result.rows;
+    res.status(201).json(user);
+  } catch (error) {
+    next(error);
+  }
+});
```

Comment:
You might eventually want an endpoint that returns just one single order, given its orderId. But you can add that one when you find out you need it. No need adding endpoints just because you _might_ need it.